cse/indexing/InvertedIndexReader.py

```diff
@@ -22,21 +22,68 @@ def close(self):
         self.__mIndex.close()
 
 
-    def retrieve(self, term):
+    def retrieveAll(self, term):

```

Comment:
Maybe we can find a better name.... What means `All`? (for now its clear but in a few weeks...)

Reply:
Yeah, I thought about that, too. But I did not come up with a good solution 😆